Source/WebCore/platform/graphics/Path.h

```diff
@@ -217,6 +217,31 @@ class Path {
 #endif
 
 private:
+    static inline constexpr auto endOfPath = static_cast<PathElement::Type>(0);
```

Comment:
Is this technically undefined behavior in C++? I wonder if it would be cleaner to just declare this as the underlying type of `PathElement::Type`.

Reply:
I thought it wasn't, but now I am unsure again. It's certainly not undefined behavior to store out of range values in the enum (as long as it's a sized enum). I will change this to comparing the underlying value.